README.md

```diff
@@ -27,13 +27,19 @@ cd tide
 yarn
 yarn start
 
-# Run storybook on your local machine
-yarn storybook
-# Deploy storybook to github pages
-yarn deploy-storybook
-
 # E2E Tests
 yarn start
 # Run this in a separate terminal
 yarn cypress open
 ```
+
+## Storybook
+
+https://tide-app.github.io/tide/
```

Comment:
Wrong url to storybook?

Reply:
Are you sure?

https://user-images.githubusercontent.com/43425812/141656270-f0dbb9a3-f558-48b8-a880-9dc6a63b2789.mov